src/core/llm/README.md

```
@@ -0,0 +1,492 @@
+// 2. Implement service
+class CustomProviderService implements ILLMService {
+    async completeTask(
+        textInput: string,
+        imageData?: ImageData,
+        fileData?: FileData,
+        stream?: boolean
+    ): Promise<string> {
+        // Custom implementation
+    }
+    
+    async getAllTools(): Promise<ToolSet> {
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)